.env

```diff
@@ -0,0 +1,12 @@
+SERVER_NAME_PROD='YOUR_PROD_SERVER_NAME'
+SERVER_NAME_LOCAL='http://localhost:3000'
+PORT=3001
```

Comment:
Is it intended that SERVER_NAME_LOCAL has a different port than PORT?

Reply:
done